### DISCUSSION TITLE
Getting "Unsupported provider: provider is not enabled" despite the provider being enabled

### DISCUSSION THREAD

**Discussion by davidgomes** [supabase/supabase#26127] [Questions] [answered]
I'm trying out Supabase Auth and I started out from this template:

https://github.com/supabase/supabase/tree/master/examples/user-management/nextjs-user-management

I deployed this to my Vercel account and then I edited the Supabase Auth settings like so:

![Screenshot 2024-05-11 at 14 45 16](https://github.com/supabase/supabase/assets/741621/5b8a6c46-f370-49cd-8ab9-d1494071976c)

I then edited the file `auth-form.tsx` to look like this:

```typescript
'use client'
import { Auth } from '@supabase/auth-ui-react'
import { ThemeSupa, supabase } from '@supabase/auth-ui-shared'
import { createClientComponentClient } from '@supabase/auth-helpers-nextjs'
import { Database } from './database.types'

export default function AuthForm() {
  const supabase = createClientComponentClient<Database>()
  return (
    <Auth
      supabaseClient={supabase}
      // view="magic_link"
      appearance={{ theme: ThemeSupa }}
      theme="dark"
      // showLinks={false}
      providers={["linkedin"]}
      // redirectTo="http://localhost:3001/auth/callback"
    />
  )
}
```

This added "LinkedIn" as an option in the UI (both in local dev and production after deploying):

![Screenshot 2024-05-11 at 14 47 37](https://github.com/supabase/supabase/assets/741621/64dde0ad-8a26-4b25-9380-577dc3b43c94)

The problem is that after clicking on it, whether in local dev or prod, I get this:

![Screenshot 2024-05-11 at 14 48 07](https://github.com/supabase/supabase/assets/741621/b76947e8-8797-4c92-a83a-38265350b2cb)

I have double and triple checked to make sure that `NEXT_PUBLIC_SUPABASE_URL` and `NEXT_PUBLIC_SUPABASE_ANON_KEY` in `.env.local` are pointing to the right Supabase project (where LinkedIn is enabled as a provider in Supabase Auth).

And this is happening both in local dev and production, again.

**Comment by GaryAustin1** [comment, marked as answer]
The provider name you are using in your OAuth call is an older version of LinkedIn provider.  For the newer OIDC version use:
![image](https://github.com/supabase/supabase/assets/54564956/dc040d69-03b0-4eab-aeed-0950bc08909b)

**Reply by davidgomes** [reply]
@GaryAustin1 thank you so much, I'll try and send a PR to @supabase/auth-ui fixing this so they have `linkedin_oidc` as a valid provider.
